lib/devices/launchpad_x.lua

```diff
@@ -0,0 +1,43 @@
+local launchpad = include('midigrid/lib/devices/generic_device')
```

Comment:
If this is the same as lauchpad_rgb can we just change this too include('midigrid/lib/devices/launchpad_rgb') ?
This will mean that once I fix some bugs and add features in launchpad_rgb in launchpad rgb the launchpad X will benefit.

Reply:
ah, yep!